draft-ietf-quic-version-negotiation.md

```diff
@@ -498,6 +498,44 @@ codepoint in the 0-63 range to replace the provisional codepoint described above
 
 --- back
 
+# Example scenarios
+
+The following is an example of compatible negotiation in the case where both endpoints support both QUIC version 2 and the version negotiation. C and S denotes client and server, respectively.
```

Comment:
QUIC version 2 doesn't exist yet. Should we use a fake version for these examples?